odsApi_versioned_docs/version-7.3/platform-dev-guide/configuration/configuration-details.mdx

```diff
@@ -0,0 +1,499 @@
+---
+sidebar_position: 1
+---
+
+# Configuration Details
+
+The Ed-Fi ODS / API is configurable in a number of ways. Since the source code
+is available to licensees, one could argue that everything is configurable. But,
+what we’re discussing in this section are things like settings and options that
+do not require a change to the compiled code.
+
+Key configuration points include: OAuth endpoints, Admin and Security datastore
+connections, Token timeouts, Enabling / disabling features. See the following
+sections for more details on important configuration options
+
+## Required Configuration Settings
+
+Some configuration must be done for every Ed-Fi ODS / API platform instance.
+Examples of required configurations include database connection strings, SMTP
+server locations, and similar.
+
+To make it easier for developers to install and run the Ed-Fi ODS / API, the
+default download from source control is pre-configured with values appropriate
+for a developer or single-server test instance of the system.
+
+### EdFi.Ods.SwaggerUI
+
+In _appsettings.json_:
```

Comment:
Formatting looks much better now. I wonder if we should just write at the top section something along the lines of - _In Ed-Fi ODS / API, appsettings.json provides the primary configuration for the ASP.NET Core applications._ - and remove this at the beginning of each section.